chapters/classes.tex

```diff
@@ -877,6 +875,12 @@ \section{Specialized Classes}\doublelabel{specialized-classes}
 \end{lstlisting}
 {]}\\ \hline
 \end{longtable}
+Additionally only components of specialized classes \lstinline!record!, \lstinline!type!, \lstinline!operator record!, and
```

Comment:
This is slightly ambiguous (in fact, I misread it at first) in that it could mean "only components of records, types, operator records, ... can be used in normal expressions". Perhaps rewording it to state, "only components which are of specialized class record, type, operator record, or connector classes based on any of these, can be used as ..."

By the way, within the source code for MapleSim, we use the word "kind" to refer to what _kind_ of class something is (model, record, etc.). We find that more concise and less ambiguous than "specialized class". Maybe it would be useful to adopt that sort of terminology in the MLS, since "class" and "type" already have well-defined meanings in the language itself.

Reply:
I implement the first proposal. The second part about "kind" is interesting, but deserves additional discussion and shouldn't be part of this PR.

Reply:
Agreed about "kind". I just took this opportunity to mention it.